pkg/collector/corechecks/net/ntp_test.go

```diff
@@ -334,16 +334,6 @@ hosts:
 	assert.Equal(t, expectedHosts, ntpCheck.cfg.instance.Hosts)
 }
 
-func TestDefaultHostConfig(t *testing.T) {
```

Comment:
Lets update this test instead to check the default behavior. We can disable all cloud provider through the `cloud_provider_metadata` configuration.